addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -1170,7 +1171,7 @@ export const formatSelection = (editor, formatName, {applyStyle, formatProps} =
     const selectedNodesInTds = [...editor.editable.querySelectorAll('.o_selected_td')]
         .map(node => closestElement(node).querySelector('br'));
     const selectedNodes = getSelectedNodes(editor.editable)
-        .filter(n => n.nodeType === Node.TEXT_NODE && closestElement(n).isContentEditable && isVisibleTextNode(n));
+        .filter(n => (n.nodeType === Node.TEXT_NODE || n.nodeName === 'BR') && closestElement(n).isContentEditable && (n.nodeName === 'BR' ? true : isVisibleTextNode(n)));
```

Comment:
How about
```suggestion
        .filter(n => ((n.nodeType === Node.TEXT_NODE && isVisibleTextNode(n)) || n.nodeName === 'BR') && closestElement(n).isContentEditable);
```
Also now we check for `BR` so we can probably revert changes done in https://github.com/odoo/odoo/pull/133188